src/firefly/tests/test_robot_gui.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from bluesky_queueserver_api import BPlan
+from ophyd import Signal
+from ophyd.sim import SynAxis
+from qtpy import QtCore
+
+from firefly.robot import RobotDisplay
+
+
+def test_region_number(qtbot):
+    """Does changing the region number affect the UI?"""
+    disp = RobotDisplay()
+    qtbot.addWidget(disp)
+    # Check that the display has the right number of rows to start with
+    # assert disp.ui.robot_combo_box.count() == 1
+    assert disp.ui.sample_combo_box.count() == 10
+    assert hasattr(disp, "regions")
+    assert len(disp.regions) == 1
+
+
+from ophyd import Component as Cpt
+
+
+class FakeHavenMotor(SynAxis):
+    user_offset = Cpt(Signal, value=0, kind="config")
+
+
+@pytest.fixture
+def sim_motor_registry(sim_registry):
+    # Create the motors
+    motor1 = FakeHavenMotor(name="motor1")
+    sim_registry.register(motor1)
```

Comment:
I don't think you need to explicitly register devices anymore. Do the tests pass without these two ``sim_registry.register()`` calls?